source/code/plugins/in_npmd_server.rb

```diff
@@ -595,6 +598,12 @@ def server_run
 
         def watch_dog_wait_for_pet
             _sleepInterval = WATCHDOG_PET_INTERVAL_SECS
+
+            # Pet right after first agent configuration is sent
+            @watch_dog_sync.synchronize do
```

Comment:
Add a small threshold on line 615 so to make sure that we go into the expected if condition as _diffTime could be = or > WATCHDOG_PET_INTERVAL_SECS